jobs/kafka/templates/config/bpm.yml

```diff
@@ -7,8 +7,7 @@ processes:
   persistent_disk: true
   env: {}
   limits:
-    memory: 1G
-    open_files: 1024
+    open_files: 131072
```

Comment:
For anyone wondering about this number, why "131072"?

Reply:
I read this.
https://kafka.apache.org/documentation/#os
> We recommend at least 100000 allowed file descriptors for the broker processes as a starting point. 

I don't think about it that much. 65536 * 2